### PR TITLE
Add Microsoft.PowerShell.MarkdownRender to PowerShell.sln

### PR DESCRIPTION
**Pull request by adityapatwardhan**
## PR Summary

#6926  caused a merge issue. Fixing the solution to retain new changes and add Microsoft.PowerShell.MarkdownRender.csproj to the solution.

## PR Checklist

- [X] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [X] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [X] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [X] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [X] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Comment by adityapatwardhan** [comment]
@bergmeister Can you also review.

**Comment by daxian-dbw** [comment]
I compared the content of `powershell.sln` from this PR with the content before #6926 was merged (see https://github.com/PowerShell/PowerShell/commits/master/PowerShell.sln), and the changes look good.

@bergmeister could you please double check?

**Review by daxian-dbw — APPROVED** [review]

**Comment by daxian-dbw** [comment]
Here is the diff between the previous correct content vs. the content in this PR:

image

I think we are good, and going to merge this PR to unblock @bergmeister.

@bergmeister Please let us know if you run into any other issues. Again, we apologize for the inconvinence.

**Comment by bergmeister** [comment]
Thanks for coming back to this

File: PowerShell.sln
Comment (bergmeister) [inline PowerShell.sln]: By comparing this to other projects, should the right hand side of this line and the line above be `CodeCoverage|Any CPU`? cc @TravisEz13

File: PowerShell.sln
Comment (bergmeister) [inline PowerShell.sln]: By comparing this to other projects, should the right hand side of this line and the line above be `Release|Any CPU`? cc @TravisEz13

File: PowerShell.sln
Comment (bergmeister) [inline PowerShell.sln]: By comparing this to other projects, should the right hand side of this line and the line above be `Release|Any CPU`? cc @TravisEz13

File: PowerShell.sln
Comment (bergmeister) [inline PowerShell.sln]: By comparing this to other projects, should the right hand side of this line and the line above be `Debug|Any CPU`? cc @TravisEz13

File: PowerShell.sln
Comment (adityapatwardhan) [reply PowerShell.sln]: I will have a look.

**Pull request merged**
